masterlist.yaml

```diff
@@ -1691,6 +1691,13 @@ plugins:
     clean:
       - crc: 0xD01056A7
         util: 'FNVEdit v4.1.5d'
+        
+  - name: 'T4-modest.esp'
```

Comment:
There's some extra space here:

```suggestion
        util: 'FNVEdit v4.1.5d'

  - name: 'T4-modest.esp'
```

Reply:
hmm why is whitespace not showing up for me?
![image](https://github.com/loot/falloutnv/assets/1944639/e9fcb2c7-7f44-4364-b7a5-aa0a1a444fba)

![image](https://github.com/loot/falloutnv/assets/1944639/9f2297ad-fffc-4171-90f9-c61b2a3fcc26)

Reply:
No clue, the GH website is weird about trailing spaces. They're definitely there though, I verified by pulling the branch locally & opening it in a text editor.

Reply:
Yeah, I can see with suggestion.
Weird, It used to show up.